workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -44,17 +43,21 @@ def chunks(g):
             yield ch
 
     s = ''
+    n = 0
```

Comment:
Maybe rename it to something like num_of_decoded_chunks.
This whole piece of code is likely to get replaced anyway soon..